Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.17
+
+WORKDIR /
+
+COPY api api
+COPY cmd cmd
+COPY pkg pkg
+COPY test test
+COPY main.go main.go
+COPY Makefile Makefile
+COPY tools.go tools.go
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+RUN make build-cli
```

Comment:
We'll want to migrate towards a multi-stage build process instead of a single stage to decrease the overall container image size. This should make it easier to reason about where this `make build-cli` throws the compiled executable as well.

Reply:
I went ahead and made a multi-stage process. What do you think?